src/usr/local/www/services_ntpd.php

```diff
@@ -487,6 +508,37 @@ function build_interface_list() {
 	'file'
 ))->addClass('btn-default');
 
+$section->addInput(new Form_Checkbox(
+	'serverauth',
+	'Enable NTP Server Authentication',
+	'Enable NTPv3 authentication (RFC 1305)',
+	$pconfig['serverauth']
+))->setHelp('Authentication allows the NTP client to verify that the server is in fact known ' .
+	    'and trusted and not an intruder intending accidentally or on purpose to masquerade as that server.');
```

Comment:
This seems a bit wordy/confusing. Could be simplified to:

> Authentication allows the NTP client to confirm it is communicating with the intended server, which protects against man-in-the-middle attacks.

Reply:
Done